src/api/schedule.py

```diff
@@ -1,3 +1,55 @@
-from flask import Blueprint
+from flask import Blueprint, request
+from webargs.flaskparser import parser
+
+from src import implementor as impl
+from src.api import schemas
+from src.api.decorators import token_required
 
 bp = Blueprint("schedule", __name__)
+
+
+@bp.route("/schedule/cole-demand", methods=["get"])
```

Comment:
Shouldn't that be coal instead of cole everywhere?

Reply:
Thanks for pointing out! Upsi! 😮